content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -168,7 +168,7 @@ export KF_DIR=${BASE_DIR}/${KF_NAME}
 
 # Set the configuration file to use, such as:
 export CONFIG_FILE=kfctl_ibm.yaml
-export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_ibm.yaml"
+export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_ibm.v1.2.0.yaml"
```

Comment:
is it not available in master?

Reply:
It's a good practice to point to a stable release, so it won't block users if something breaks in master.

Reply:
also it gives a better sense for us to know what version our user is deploying